api/apps/api/src/modules/scenarios/scenarios.controller.ts

```diff
@@ -671,4 +678,36 @@ export class ScenariosController {
     }
     return AsyncJobDto.forScenario().asJsonApiMetadata();
   }
+
+  @ApiOkResponse({
+    type: JsonApiAsyncJobMeta,
+  })
+  @Post(`:id/calibration`)
+  async startCalibration(
+    @Param('id', ParseUUIDPipe) id: string,
+    @Body() { range }: StartScenarioBlmCalibrationDto,
+  ): Promise<JsonApiAsyncJobMeta> {
+    const result = await this.service.startBlmCalibration(id, range);
+
+    if (isLeft(result)) {
+      switch (result.left) {
+        case planningUnitAreaNotFound:
+          throw new NotFoundException(
+            `Could not found planning units area for scenario with ID: ${id}`,
+          );
```

Comment:
For external user, this could actually be `internal server error` - as if we couldn't find PU area, we did something really wrong 🤔

Reply:
Replaced `NotFoundException` with `InternalServerErrorException`